src/main/java/org/mule/module/json/internal/ValidateJsonSchemaOperation.java

```diff
@@ -131,7 +133,9 @@ public void stop() {
   @Validator
   @Execution(CPU_INTENSIVE)
   @Throws(SchemaValidatorErrorTypeProvider.class)
-  public void validateSchema(@Summary("The schema location") @Path(type = FILE, acceptedFileExtensions = "json") String schema,
+  public void validateSchema(@Optional @Summary("The schema location") @Path(type = FILE,
+      acceptedFileExtensions = "json") String schema,
```

Comment:
In this case, I think there is no problem adding parameters in a public method because it is an internal package. But, this kind of change breaks backward compatibility when the class is exposed as API.

Reply:
In this case we choose to use a grouping through ParameterGroup and to be able to handle mutual exclusion at design time and also avoid breaking compatibility